Regression: System.Runtime.Tests fail with "Missing a required constructor" on System.Reflection.PseudoCustomAttribute._GetSecurityAttributes

call CoreRun.exe xunit.console.netcore.exe System.Runtime.Tests.dll  -xml testResults.xml -notrait Benchmark=true -notrait category=nonnetcoreapp1.1tests  -notrait category=OuterLoop -notrait category=failing -notrait category=nonwind
  owstests
  xUnit.net console test runner (64-bit .NET Core)
  Copyright (C) 2014 Outercurve Foundation.

  Discovering: System.Runtime.Tests
  Discovered:  System.Runtime.Tests
  Starting:    System.Runtime.Tests
     System.Tests.GetCustomAttribute.NegTest1 [FAIL]
        Assert.Throws() Failure
        Expected: typeof(System.Reflection.AmbiguousMatchException)
        Actual:   typeof(System.Runtime.InteropServices.COMException): Missing a required constructor. (Exception from HRESULT: 0x8013143B)
        Stack Trace:
              at System.Reflection.PseudoCustomAttribute._GetSecurityAttributes(RuntimeModule module, Int32 token, Boolean assembly, Object[]& securityAttributes)
              at System.Reflection.PseudoCustomAttribute.GetCustomAttributes(RuntimeAssembly assembly, RuntimeType caType, Boolean includeSecCa, Int32& count)
              at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeAssembly assembly, RuntimeType caType)
              at System.Attribute.GetCustomAttributes(Assembly element, Type attributeType, Boolean inherit)
              at System.Attribute.GetCustomAttribute(Assembly element, Type attributeType, Boolean inherit)
           D:\git\corefx\src\System.Runtime\tests\System\Attributes.netstandard1.7.cs(256,0): at System.Tests.GetCustomAttribute.<>c__DisplayClass2_0.<NegTest1>b__3()
     System.Tests.GetCustomAttribute.NegTest2 [FAIL]
        Assert.Throws() Failure
        Expected: typeof(System.Reflection.AmbiguousMatchException)
        Actual:   typeof(System.Runtime.InteropServices.COMException): Missing a required constructor. (Exception from HRESULT: 0x8013143B)
        Stack Trace:
              at System.Reflection.PseudoCustomAttribute._GetSecurityAttributes(RuntimeModule module, Int32 token, Boolean assembly, Object[]& securityAttributes)
              at System.Reflection.PseudoCustomAttribute.GetCustomAttributes(RuntimeAssembly assembly, RuntimeType caType, Boolean includeSecCa, Int32& count)
              at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeAssembly assembly, RuntimeType caType)
              at System.Attribute.GetCustomAttributes(Assembly element, Type attributeType, Boolean inherit)
              at System.Attribute.GetCustomAttribute(Assembly element, Type attributeType, Boolean inherit)
           D:\git\corefx\src\System.Runtime\tests\System\Attributes.netstandard1.7.cs(298,0): at System.Tests.GetCustomAttribute.<>c__DisplayClass4_0.<NegTest2>b__3()
     System.IO.Tests.FileLoadExceptionInteropTests.Fom_HR(hr: -2146232799) [FAIL]
        Assert.Equal() Failure
        Expected: (null)
        Actual:   complib.hlp#5179
        Stack Trace:
           D:\git\corefx\src\System.Runtime\tests\System\IO\Exceptions.Utility.cs(23,0): at System.IO.Tests.ExceptionUtility.ValidateExceptionProperties(Exception e, Int32 dataCount, String helpLink, Int32 hResult, Exception innerExcept
  ion, String message, String source, String stackTrace, Boolean validateMessage)
           D:\git\corefx\src\System.Runtime\tests\System\IO\FileLoadException.InteropTests.cs(46,0): at System.IO.Tests.FileLoadExceptionInteropTests.Fom_HR(Int32 hr)
     System.Tests.AttributeGetCustomAttributes.RunPosTests [FAIL]
        System.Runtime.InteropServices.COMException : Missing a required constructor. (Exception from HRESULT: 0x8013143B)
        Stack Trace:
              at System.Reflection.PseudoCustomAttribute._GetSecurityAttributes(RuntimeModule module, Int32 token, Boolean assembly, Object[]& securityAttributes)
              at System.Reflection.PseudoCustomAttribute.GetCustomAttributes(RuntimeAssembly assembly, RuntimeType caType, Boolean includeSecCa, Int32& count)
              at System.Reflection.CustomAttribute.GetCustomAttributes(RuntimeAssembly assembly, RuntimeType caType)
              at System.Attribute.GetCustomAttributes(Assembly element, Boolean inherit)
           D:\git\corefx\src\System.Runtime\tests\System\Attributes.netstandard1.7.cs(122,0): at System.Tests.AttributeGetCustomAttributes.RunPosTests()
  Finished:    System.Runtime.Tests

  === TEST EXECUTION SUMMARY ===
D:\git\corefx\Tools\tests.targets(260,5): warning :    System.Runtime.Tests  Total: 10501, Errors: 0, Failed: 4, Skipped: 0, Time: 1.910s [D:\git\corefx\src\System.Runtime\tests\System.Runtime.Tests.csproj]
  The previous error was converted to a warning because the task was called with ContinueOnError=true.
  Finished running tests.  End time= 9:36:42.45, Exit code = 4
D:\git\corefx\Tools\tests.targets(260,5): warning MSB3073: The command "D:\git\corefx\bin/tests/AnyOS.AnyCPU.Debug/System.Runtime.Tests/default.netcoreapp1.1/netcoreapp1.1/RunTests.cmd D:\git\corefx\packages/" exited with code 4. [D:\gi
t\corefx\src\System.Runtime\tests\System.Runtime.Tests.csproj]

--- a/src/System.Runtime/tests/System/Attributes.netstandard1.7.cs
+++ b/src/System.Runtime/tests/System/Attributes.netstandard1.7.cs
@@ -112,7 +112,6 @@ namespace System.Tests
         }
 
         [Fact]
-        [ActiveIssue(12759)]
         public static void RunPosTests()
         {
             Type clsType2 = typeof(TestClass2);
@@ -240,7 +239,6 @@ namespace System.Tests
         }
 
         [Fact]
-        [ActiveIssue(12759)]
         public static void NegTest1()
         {
             Assembly element = null;
@@ -283,7 +281,6 @@ namespace System.Tests
         }
 
         [Fact]
-        [ActiveIssue(12759)]
         public static void NegTest2()
         {
             Assembly element = null;

--- a/src/System.Runtime/tests/TestAssembly/project.json
+++ b/src/System.Runtime/tests/TestAssembly/project.json
@@ -9,6 +9,7 @@
   "frameworks": {
     ".NETCoreApp,Version=v1.0": {},
     ".NETCoreApp,Version=v1.1": {},
-    "netstandard1.3": {}
+    "netstandard1.3": {},
+    "netstandard1.5": {}
   }
 }

